test/integration/bpmnlint-plugin-test2/index.js

```diff
@@ -7,5 +7,8 @@ module.exports = {
         'bpmnlint/no-disconnected': 'warn'
       }
     }
+  },
+  rules: {
+    'no-label-xxx': require('./lib/rules/no-label-xxx')
```

Comment:
Are we removing a test case here?

Proposal: Let's rather introduce another rule, i.e. `custom-location`.